dnsserv_test.go

```diff
@@ -1,40 +0,0 @@
-package main
-
-import (
-	"bytes"
-	. "gopkg.in/check.v1"
-	"net"
-	"testing"
-)
-
-func Test(t *testing.T) { TestingT(t) }
-
-type DNSNameserverSuite struct{}
-
-var _ = Suite(&DNSNameserverSuite{})
-
-func (s *DNSNameserverSuite) TestUDPConnection(c *C) {
```

Comment:
The first test was testing a function that we didn't implement ourselves (net.SplitHostPort( )). And we don't call this function anymore. We could write a test assert on what getUDPAddr( ) returns but how much value is there in asserting that the server will listen on a specific address and port?

The last two tests were asserting on the values returned by a function that isn't defined anymore with this commit.